clients/admin-ui/src/pages/management/cors-configuration.tsx

```diff
@@ -0,0 +1,172 @@
+/* eslint-disable react/no-array-index-key */
+import { AddIcon } from "@chakra-ui/icons";
+import {
+  Box,
+  Button,
+  DeleteIcon,
+  Flex,
+  Heading,
+  IconButton,
+  Link,
+  Text,
+  useToast,
+} from "@fidesui/react";
+import { SerializedError } from "@reduxjs/toolkit";
+import { FetchBaseQueryError } from "@reduxjs/toolkit/dist/query";
+import { FieldArray, Form, Formik, FormikHelpers } from "formik";
+import type { NextPage } from "next";
+
+import { useAppSelector } from "~/app/hooks";
+import FormSection from "~/features/common/form/FormSection";
+import { CustomTextInput } from "~/features/common/form/inputs";
+import { getErrorMessage, isErrorResult } from "~/features/common/helpers";
+import Layout from "~/features/common/Layout";
+import { errorToastParams, successToastParams } from "~/features/common/toast";
+import {
+  selectCORSDomains,
+  useCreateConfigurationSettingsMutation,
+  useGetConfigurationSettingsQuery,
+} from "~/features/privacy-requests/privacy-requests.slice";
+
+type FormValues = {
+  domains: string[];
+};
+
+const CORSConfigurationPage: NextPage = () => {
+  const { isLoading: isLoadingGetQuery } = useGetConfigurationSettingsQuery();
+  const corsDomains = useAppSelector(selectCORSDomains());
```

Comment:
I'm kind of surprised this works, I thought it had to be `useAppSelector(selectCORSDomains)`!

Reply:
I _think_ I did this because it didn't work without the `()`. I could be incorrect on that. Either way it still works with the function being called 🤷‍♂️